platforms/corda-ent/charts/bridge/values.yaml

```diff
@@ -0,0 +1,119 @@
+  # e.g. name: nmap
+  name: nmap
+  # Provide the port where the Networkmap service is exposed.
+  # e.g. port: 10000
+  port: 10000
+  # Provide the namespace where the Networkmap is deployed.
+  namespace: baah
+
+# Provide context for the chart.
+metadata:
+  # Provide the namespace where the Bridge will be deployed.
+  # e.g. namespace: cenm
+  namespace: baah
+  # Provide any extra labels you wish to add to the charts, formatted as key-value pairs.
+  # e.g. key: "value" 
+  labels: {}
+
+# Provide the amount of replicas you want to deploy of the Float.
+# e.g. replicas: 1
+replicas: 1
+
+
+image:
+  initContainerName: index.docker.io/hyperledgerlabs/alpine-utils:1.0
+  imagepullsecret: regcred
+
+# Information about the Docker container used for the init-containers.
+initContainerImage:
+  # Provide the name of the image, including the tag.
+  # e.g. name: index.docker.io/hyperledgerlabs/alpine-utils:1.0
+  name:
+
+# Information about the main image used for the main Float firewall container.
+image:
+  # Provide the name of the image, including the tag
+  # e.g. name: adopblockchaincloud0502.azurecr.io/corda_image_firewall_4.4:latest
+  name:
+  # Provide the K8s secret that has rights for pulling the image off the registry.
+  # NOTE: Make sure the secret exists prior to manually running this deployment, 
+  # as this chart does not create the pull secret itself.
+  # e.g. pullSecret: regcred
+  pullSecret: 
+  # Provide the pull policy for Docker images, either Always or IfNotPresent.
+  # e.g. pullPolicy: Always
+  pullPolicy:
+
+
+# Information about the vault.
+# NOTE: Make sure that the vault is already unsealed, initialized and configured
+# to use the Kubernetes service account token based authentication.
+vault:
+  # Provide the vault address
+  # Eg. address: http://vault.example.com:8200
+  address: http://vault.internal.dev.aws.blockchaincloudpoc-develop.com:8200
+  # Provide the vault role used.
+  # Eg. role: vault-role 
+  role: vault-role
+  # Provide the authpath configured to be used.
+  # Eg. authpath: entcordacenm
+  authpath: entcordacenm
+  # Provide the service account name autheticated to vault.
+  # NOTE: Make sure that the service account is already created and autheticated to use the vault.
+  # Eg. serviceaccountname: vault-auth
+  serviceaccountname: vault-auth
+  # Provide the vault path where the  certificates are stored
+  # Eg. certsecretprefix: cenm/certs
+  certsecretprefix: cenm/certs
+
+
+
+# Provide volume related specifications
+volume:
+  # Provide the base directory for the container.
+  # E.g. baseDir: /opt/corda
+  baseDir: /opt/corda
+
+# This section contains information about storage classes used in the Chart.
+storage:
+  # Provide the name of the storage class. 
+  # NOTE: Make sure that the storage class exists prior to manually running this deployment, 
+  # as this chart does not create the storage class itself.
+  # e.g. name: cenm
+  name: abcd
```

Comment:
Please remove actual values from value.yaml file

Reply:
Fixed in [ec7e5a7](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/897/commits/ec7e5a75829cd0887cdedb55ee0c60e19266150b)